src/pages/ServiceDetails/__tests__/__snapshots__/ServiceInfo.test.tsx.snap

```diff
@@ -389,13 +389,6 @@ ShallowWrapper {
                       eventKey="destinationrules"
                     >
                       Destination Rules (1)
-                      <span>
```

Comment:
I see in this snapshot that now validation icons are not shown for destination rules tab. I think it is because of [this](https://github.com/xeviknal/kiali-ui/blob/a624c044ecbd8d71e795d5bba349c36847ce9a15/src/services/__mockData__/getServiceDetail.json) request mock hasn't been updated. The feature is working well though.